cli/features/tutorial.feature

```diff
@@ -28,7 +28,7 @@ Feature: Following the tutorial
       | Name of the application to create | todo-app           |
       | Description                       | A todo application |
       | Initial version                   |                    |
-      | ExoCom version                    | 0.16.0             |
+      | ExoCom version                    | latest             |
```

Comment:
We can, but each time we update ExoCom we'd have to change the version number in the feature tests and all of the example application templates.

It seemed hacky to me when I had to go through the whole monorepo to find and replace `0.16.0` with `0.16.1`. Plus a new version of ExoCom could be released and our tests would be running against an older version. The developer can certainly define which version of ExoCom they want in their application.yml though.